composer.json

```diff
@@ -17,11 +17,11 @@
     ],
 
     "require": {
-        "php": "^5.5.9|^7.0.8|^7.1.3|^7.2.5",
-        "symfony/http-foundation": "~3.0|~4.0|~5.0"
+        "php": "^7.2",
+        "symfony/http-foundation": "^4.4|^5.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.8 || ^5.0"
+        "phpunit/phpunit": "^8.5"
```

Comment:
I think we should rather use `symfony/phpunit-bridge` to avoid bumping PHP version.

This would also eliminate the need of many other changes here.